app/lib/controlled_work_mappings/cw1_welsh_new.yml

```diff
@@ -0,0 +1,436 @@
+
+    # Pensions / Client
+    - name: pensions_client
+      type: text
+      source: client_pensions
+
+    # Student Finance / Client
+    - name: studentfinance_client
+      type: text
+      source: client_student_finance
+
+    # Family or Friends / Client
+    - name: friendsandfamily_client
+      type: text
+      source: client_friends_and_family
+
+    # Property or Lodger / Client
+    - name: property_client
+      type: text
+      source: client_property_lodger_income
+
+    # Other Income / Client
+    - name: otherincome_client
+      type: text
+      source: client_other_income
+
+    # Benefits & Tax Credits / Client
+    - name: benefits_client
+      type: text
+      source: client_benefits
+
+    # Other income / Client
+    - name: FillText152
+      type: text
+      source: client_non_employment_income
+
+    # Total gross income / Client
+    - name: FillText154
+      type: text
+      source: client_total_income
+
+    # Total gross income  Client and Partner
+    - name: FillText164
+      type: text
+      section: partner_income
+      source: total_combined_income
+
+    # Mortgage instalment / Client
+    - name: FillText2
+      type: text
+      source: client_mortgage
+
+    # Rent / Client
+    - name: FillText6
+      type: text
+      source: client_rent
+
+    # Dependents' allowances / Partner
+    - name: FillText9
+      type: text
+      source: partner_allowance
+
+    # Dependents' allowances / Aged 15 or under
+    - name: FillText10
+      type: text
+      source: dependants_allowance_under_16
+
+    # Dependents' allowances / Aged 16 or over
+    - name: FillText11
+      type: text
+      source: dependants_allowance_over_16
+
+    # Tax and national insurance / Client
+    - name: FillText22
+      type: text
+      source: client_tax_and_national_insurance
+
+    # Standard allowance for employment expense / Client
+    - name: FillText12
+      type: text
+      source: client_employment_deduction
+
+    # Maintenance allowance / Client
+    - name: FillText14
+      type: text
+      source: client_maintenance_allowance
+
+    # Childcare costs
+    - name: FillText17
+      type: text
+      source: combined_childcare_costs
+
+    # Payment of income contribution order / Client
+    - name: FillText32
+      type: text
+      source: client_legal_aid_contribution
+
+    # Total allowances / Client
+    - name: FillText18
+      type: text
+      source: client_total_allowances
+
+    # Total monthly disposable income / Client
+    - name: FillText20
+      type: text
+      source: client_disposable_income
+
+    # Total monthly disposable income Client and partner
+    - name: FillText31
+      type: text
+      source: combined_disposable_income
+
+
+- section: partner_income
+  fields:
+
+    # Gross monthly earnings / Partner
+    - name: FillText151
+      type: text
+      source: partner_gross_income
+
+    # Other income / Partner
+    - name: FillText153
```

Comment:
I think we've missed the partner income mappings.

`FillText153` no longer exists, we instead need to add in:

- benefits_partner
- maintenance_partner
- pensions_partner
- studentfinance_partner
- friendsandfamily_partner
- property_partner
- otherincome_partner